StartEmailValidationProcess/orchestrators.ts

```diff
@@ -12,10 +12,13 @@ import { EmailAddress } from "../generated/backend/EmailAddress";
 import { toHash } from "../utils/crypto";
 
 /**
+            // eslint-disable-next-line jsdoc/newline-after-description
```

Comment:
```suggestion
   *
```

Reply:
removed also this king of flag since they are not useful anymore.

Reply:
done in https://github.com/pagopa/io-functions-app/pull/211/commits/cdd35b20738d5816d008c3d2665a1b745ce5fdca